packages/shared/src/components/post/PostActions.tsx

```diff
@@ -75,50 +87,89 @@ export function PostActions({
     return undefined;
   };
 
+  const toggleDownvote = () => {
+    if (!post) {
+      return;
+    }
+
+    if (!user) {
+      showLogin(AuthTriggers.Downvote);
+
+      return;
+    }
+
+    if (post.downvoted) {
+      cancelPostDownvote({ id: post.id });
+    } else {
+      downvotePost({ id: post.id });
+    }
+  };
+
   return (
-    <div className="flex justify-between py-2 px-4 rounded-16 border border-theme-divider-tertiary">
-      <QuaternaryButton
-        id="upvote-post-btn"
-        pressed={post.upvoted}
-        onClick={toggleUpvote}
-        icon={<UpvoteIcon secondary={post.upvoted} />}
-        aria-label="Upvote"
-        responsiveLabelClass={actionsClassName}
-        className="btn-tertiary-avocado"
-      >
-        Upvote
-      </QuaternaryButton>
-      <QuaternaryButton
-        id="comment-post-btn"
-        pressed={post.commented}
-        onClick={onComment}
-        icon={<CommentIcon secondary={post.commented} />}
-        aria-label="Comment"
-        responsiveLabelClass={actionsClassName}
-        className="btn-tertiary-blueCheese"
-      >
-        Comment
-      </QuaternaryButton>
-      <QuaternaryButton
-        id="bookmark-post-btn"
-        pressed={post.bookmarked}
-        onClick={onBookmark}
-        icon={<BookmarkIcon secondary={post.bookmarked} />}
-        aria-label="Bookmark"
-        responsiveLabelClass={actionsClassName}
-        className="btn-tertiary-bun"
-      >
-        Bookmark
-      </QuaternaryButton>
-      <QuaternaryButton
-        id="share-post-btn"
-        onClick={() => onShare(post)}
-        icon={<ShareIcon />}
-        responsiveLabelClass={actionsClassName}
-        className="btn-tertiary-cabbage"
+    <div className="flex items-center rounded-16 border border-theme-divider-tertiary">
+      <Card
+        className={classNames(
+          'flex !flex-row hover:border-theme-divider-tertiary gap-2',
+          {
+            'border-theme-color-avocado hover:!border-theme-color-avocado bg-avocado-50/8':
+              post.upvoted,
+            'border-theme-color-ketchup hover:!border-theme-color-ketchup bg-ketchup-50/8':
```

Comment:
After verifying in Sketch, it seems the value should be `-40`. Reference: https://github.com/dailydotdev/apps/pull/1880#discussion_r1239258989

Just not sure if it would work the same with the opacity.
```suggestion
            'border-theme-color-ketchup hover:!border-theme-color-ketchup bg-theme-color-ketchup/8':
```